internal/security/setup/option/generate.go

```diff
@@ -57,40 +64,47 @@ func isGenerateNoOp(pkiInitOption *PkiInitOption) bool {
 }
 
 func generatePkis() (exitCode, error) {
-	baseWorkingDir, err := os.Getwd()
+	certConfigDir, err := getCertConfigDir()
 	if err != nil {
 		return exitWithError, err
 	}
 
-	resourceDirPath := filepath.Join(baseWorkingDir, resourceDirName)
-	pkiSetupVaultJSONPath := filepath.Join(resourceDirPath, pkiSetupVaultJSON)
-	pkiSetupKongJSONPath := filepath.Join(resourceDirPath, pkiSetupKongJSON)
+	certConfigDir, err = filepath.Abs(certConfigDir)
+	if err != nil {
+		return exitWithError, err
+	}
+	pkiSetupVaultJSONPath := filepath.Join(certConfigDir, pkiSetupVaultJSON)
```

Comment:
it would be nice if there was a way to specify the actual filename through configuration.toml, but that's not necessary for this PR, but please file an issue for doing that later on. Probably for Geneva or later I think, it's low priority.

Reply:
+1 for future enhancement

Reply:
created an issue for this one. Don't see a way to label it for Geneva, @anonymouse64 ir @jim-wang-intel, could you label it?
https://github.com/edgexfoundry/edgex-go/issues/1885

Reply:
done